Dockerfile

```
@@ -7,45 +7,41 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y software-properties-common && add-apt-repository -y ppa:mhier/libboost-latest
 
 FROM base as builder
```

This will remove a warning (`WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match`) during the build:

```suggestion
FROM base AS builder
```